### PR TITLE
Internationalization for flock-registration

### PR DESCRIPTION
**Pull request by echevemaster**
I'm thinking use flock-registration for spanish-speakers events. So, i'm translating the application via babel, let me know if you want to introduce this feature to upstream, missing some corrections to the templates, and corresponding translations.

### CODE REVIEW

File: wsgi/registration.py [hunk 1]
```diff
@@ -181,7 +195,7 @@ def new():
         except:
             pass
     return flask.render_template('registration.html', form=form,
-                                 submit_text="Submit registration")
+                                 submit_text=gettext("Submit registration"))
```
Comment (yograterol) [inline wsgi/registration.py:226]: @echevemaster, you should have a convention about quotes and single quotes

**Comment by lmacken** [comment]
Thank you for the patch! I definitely would love to merge this upstream.

Can you please do me a favor and `git pull --rebase` and repush so GitHub will be able to automatically mere this PR? Thanks!

**Comment by yograterol** [comment]
I agree with @lmacken, @echevemaster BTW where is the locale files for both languages?

**Comment by echevemaster** [comment]
@lmacken , done. 
commit 177a89b have a tool based on Flask-Script for manage the database tasks.

**Comment by echevemaster** [comment]
@yograterol I have to make the translations, I will push ASAP. 

if you want to help with the task, please run in wsgi directory: 
pybabel extract -F babel.cfg -k lazy_gettext -o messages.pot .

**Comment by lmacken** [comment]
Thanks! :thumbsup:

**Pull request merged**